Containerfile

```diff
@@ -1,17 +1,52 @@
-FROM registry.access.redhat.com/ubi9/ubi-minimal AS build
+FROM registry.fedoraproject.org/fedora:latest AS build_python
 
-RUN \
-    microdnf install -y \
+RUN dnf install -y \
         python3-pip \
         python3-pyyaml \
-        && microdnf clean all
+        && dnf clean all
 
 WORKDIR /src
 
 COPY pyproject.toml /src/
 COPY src /src/src
 
 RUN pip install --no-dependencies .
-RUN rm -rf src/*
+
+# ----
+FROM registry.access.redhat.com/ubi9/go-toolset:latest AS build_go
+
+ARG CONTAINERS_STORAGE_THIN_TAGS="containers_image_openpgp exclude_graphdriver_btrfs exclude_graphdriver_devicemapper"
+ENV CONTAINERS_STORAGE_THIN_TAGS=${CONTAINERS_STORAGE_THIN_TAGS}
+
+ARG IMAGES_REF="github.com/osbuild/images"
+ENV IMAGES_REF=${IMAGES_REF}
+
+# Note that "external" will most likely in the future build from internal
+# sources instead of pulling off the network
+RUN sh -c 'for otk_cmd in gen-partition-table \
```

Comment:
Silly question, could we just run "make externals" or something here to avoid the duplication of this external list (and the exclude tags)?

Reply:
Sure, thanks for the hint 🥳, I would have optimized this once it's fully working (see my last comment) and I got the confirmation that we should even package this into the container (in contrast to bind-mounting it)

Where I think a container which is fully working (including all externals we know) is beneficial to get started faster

Reply:
Oh, sorry, then my comment was a bit premature I guess (sry!)